### PR TITLE
Contact form finish mobile version

### PR DESCRIPTION
**Pull request by Zuwairanajma**
Here I:
-Added contact form elements.
-Styled the elements according to the Figma template.
-Created a new branch and pushed it to my repository.

### CODE REVIEW

**Comment by Zuwairanajma** [comment]
Announcement from Microverse official:
Hey Micronauts @channel! :wave:
I hope you're doing well! We are aware that most of you are experiencing Webhint issues, we are currently working on a solution:male-technologist:.
For now, please go ahead and request for review if you only have the Webhint error on your PR. Our CRs would be happy to review your PRs :slightly_smiling_face:.
Feel free to share this announcement on your PR for the CR to see:bangbang:
Thank you for your understanding.

Marco Lizza

File: styles.css [hunk 1]
```diff
@@ -654,6 +654,154 @@ li {
...
+  border: 1px solid transparent;
+  line-height: 24px;
+  font-size: 17px;
+  padding: 12px;
+  font-weight: 700;
+  background-color: #f55800;
+  color: #dbd8d7;
+  margin-left: 100px;
+  margin-top: 0;
+}
```
Comment (AbelG101) [inline styles.css]: - [ ] Ouch! this is close but you seem to forget to implement the `:hover` & `:active` interactions for the "Get in touch" button, as a frontend developer you have to pay attention to small details like this as they provide a polished look to your UI. 😉
- Here's the Figma design as a reminder:
  firefox_PKDzK8XvCT
- [ ] Also make sure to adjust the color of the text in the button as it should be white initially and not `#dbd8d7`.

File: styles.css [hunk 2]
```diff
@@ -654,6 +654,154 @@ li {
...
+  flex-direction: column;
+  justify-content: inline-block;
+}
+
+.hori-div {
+  background-color: gray;
+  width: 375px;
+  margin-top: -10px;
+  align-items: center;
+}
```
Comment (AbelG101) [inline styles.css]: - [ ] Kindly make sure to stay consistent with the given Figma design when choosing colors as I noticed the background color of the horizontal line in your design should be `#DFE1E6` and not gray so make sure to fix this.
- Your design:
  image
- Figma design:
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Zuwairanajma 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Form is properly validated✔
- The UI is looking fabulous, keep it up! ✔
- Descriptive PR title with a good summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @Zuwairanajma, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
*  Good use of GitHub flow ✔️
*  Meaningful commit messages ✔️
*  Brilliant implementation ✔️
## Optional suggestions
* N/A
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**